pydatastructs/miscellaneous_data_structures/queue.py

```diff
@@ -51,6 +51,11 @@ def __new__(cls, implementation='array', **kwargs):
             return ArrayQueue(
                 kwargs.get('items', None),
                 kwargs.get('dtype', int))
+        elif implementation == 'linkedlist':
+            return LinkedListQueue(
+                # kwargs.get('items', None),
+                # kwargs.get('dtype', LinkedListNode)
```

Comment:
Why is this commented out? Can we keep the API similar to the `ArrayQueue`?

Reply:
Made `LinkedListQueue` analogous to `ArrayQueue`. Now they have similar API.